meson.build

```diff
@@ -0,0 +1,820 @@
+# TODO: cross-compile supports is rudimentary
+
+project('uftrace', 'c', version: '0.10')
+python_module = import('python')
+
+common_cflags = ['-W', '-Wall', '-D_GNU_SOURCE', '-D_DEFAULT_SOURCE', '-D_XOPEN_SOURCE=600']
+common_cflags += '-Wdeclaration-after-statement'
+common_cflags += '-Wno-unused-parameter'
+common_cflags += '-Wno-missing-field-initializers'
+common_ldflags = []
+uftrace_cflags    = ['-DINSTALL_LIB_PATH="@0@"'.format(get_option('prefix') / get_option('libdir'))]
+libmcount_cflags  = ['-fvisibility=hidden', '-fno-omit-frame-pointer']
+demangler_cflags  = []
+symbols_cflags    = []
+dbginfo_cflags    = []
+traceevent_cflags = []
+test_cflags       = []
+
+program_shell = find_program('sh', required: true)
+program_git = find_program('git', required: true)
+
+cc = meson.get_compiler('c')
+
+host_machine_cpu_family = host_machine.cpu_family()
+if host_machine_cpu_family == 'x86'
+    host_machine_cpu_family = 'i386'
+endif
+
+if get_option('arch') == 'auto'
+    arch = host_machine_cpu_family
+elif get_option('arch') == host_machine_cpu_family
+    arch = host_machine_cpu_family
+else
+    error('host compiler `@0@` doesn\'t match host architecture `@1@`'.format(
+        host_machine_cpu_family, get_option('arch')))
+endif
+
+################################
+# additional compilation flags #
+################################
+
+if get_option('debugging')
+    common_cflags += ['-O0', '-g']
+else
+    common_cflags += ['-O2', '-g']
+endif
```

Comment:
Sorry, we need another update again because of the recent change at [here](https://github.com/namhyung/uftrace/commit/76027ad368a3d2ec2485d9a1e52ac8078e05cbff?branch=76027ad368a3d2ec2485d9a1e52ac8078e05cbff&diff=unified#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52). Could you please apply this change?
```diff
diff --git a/meson.build b/meson.build
index 04055ba4..98dd0c5a 100644
--- a/meson.build
+++ b/meson.build
@@ -40,9 +40,11 @@ endif
 ################################

 if get_option('debugging')
-    common_cflags += ['-O0', '-g']
+    common_cflags += ['-O0', '-g', '-DDEBUG_MODE=1']
+    common_ldflags += ['-lunwind']
 else
-    common_cflags += ['-O2', '-g']
+    common_cflags += ['-O2', '-g', '-DDEBUG_MODE=0']
+    common_ldflags += ['-rdynamic']
 endif

 if get_option('trace')
```